src/sass/components/_section-gallery.scss

```diff
@@ -0,0 +1,364 @@
+                    }
+                  }
+                }
+              }
+
+              .details-circle {
+                height: 100px;
+                width: 100px;
+                background-color: rgba(123, 123, 124, 0.8);
+                border-radius: 50%;
+                position: absolute;
+                transform: translate(0, -110%);
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                align-items: center;
+
+                .price-new {
+                  font-size: 18px;
+                  margin: 0;
+                  color: white;
+                  font-weight: bold;
+                  letter-spacing: 1px;
+                }
+
+                .price-old {
+                  display: inline-flex;
+                  text-decoration: line-through;
+                  color: white;
+                  margin: 0;
+                  font-size: 13px;
+                  letter-spacing: 1px;
+                }
+              }
+            }
+          }
+        }
+
+        .gallery-slider {
+          width: 100%;
+          height: 88px;
+          display: flex;
+          justify-content: space-around;
+          align-items: center;
+          border: 2px solid #e2e2e2;
+          padding: 0 5px;
+        }
+
+        .slider-box {
+          height: 70px;
+          width: 30px;
+          background-color: #2a2a2a;
+          color: white;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+        }
+
+        .slider-box-arr {
+          &:hover {
+            cursor: pointer;
+            color: #d58e32;
+          }
+        }
+
+        .pictures-area {
+          width: 100%;
+          height: 88px;
+          display: flex;
+          justify-content: space-around;
+          align-items: center;
+
+          .slider-image {
+            width: 70px;
+            background-color: #cccccc;
+
+            &:hover {
+              cursor: pointer;
+            }
+
+            .js-image {
+              height: 100%;
+              width: 100%;
+              object-fit: contain;
+              overflow: hidden;
+            }
+
+            .js-image-ovl {
+              opacity: 0.5;
+            }
+          }
+        }
+
+        .slider-active {
+          opacity: 1;
+        }
+      }
+    }
+
+    .big-picture-area {
+      height: 600px;
+      background-color: #e8edf7;
+      width: 50%;
+      position: relative;
+
+      img {
+        height: 100%;
+        width: 100%;
+        background-position: center;
+        background-size: cover;
+        overflow: hidden;
+        object-fit: cover;
+      }
+
+      .text-area {
+        position: absolute;
+        top: 50px;
+        left: 50px;
+        display: flex;
+        flex-direction: column;
+
+        .text-price {
+          display: flex;
+          align-items: flex-end;
+          text-transform: uppercase;

```

Comment:
Do przedyskutowania, czy klient powinien wpisywać dużymi literami, czy ustawić w kodzie.
To rozwiązanie ma tą wadę, że jak klient wpisze małymi literami, a pojawi się dużymi na stronie, to będą pretensje dlaczego nie działa